src/setu.js

```diff
@@ -50,6 +50,15 @@ class Setu {
         }
     };
 
+    setBillExpiry(platformBillID=null) {
+        if (this.checkIfValuesExist()) {
```

Comment:
verify is platformBillID exists as below

Reply:
done